.github/workflows/build.yml

```
@@ -264,7 +267,10 @@ jobs:
   # ── aarch64 native build ──────────────────────────────────────────────────
   # Mirrors the x86_64 build job. ARM failures do not fail the workflow or
   # block x86_64 publication — this is an experimental parallel build.
+  # Disabled: webkit cold-builds OOM/timeout the ARM runner. Re-enable once
+  # the webkit artifacts are cached in cache.projectbluefin.io.
   build-aarch64:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)